Resources/Prototypes/Datasets/ion_storm.yml

```diff
@@ -243,18 +239,22 @@
   - THE EMERGENCY SHUTTLE
   - THE ESCAPE PODS
   - THE GALAXY
-  - THE GULAG
```

Comment:
A gulag is kinda ultra-specific so not really likely to be funny

Reply:
Could be "Mining Colony on Mars"